command/connect/redirecttraffic/redirect_traffic_test.go

```diff
@@ -549,6 +549,12 @@ func TestGenerateConfigFromFlags(t *testing.T) {
 		t.Run(c.name, func(t *testing.T) {
 			cmd := c.command()
 			if c.consulServices != nil {
+				// Skip test when -short flag provided; any tests that create a test server
+				// will take at least 100ms which is undesirable for -short
+				if testing.Short() {
+					t.Skip("too slow for testing.Short")
+				}
```

Comment:
I believe this function `TestGenerateConfigFromFlags` already has this `t.Skip` logic as the first line of the test. I believe it was only added recently, but it seems to be working. So I suspect adding the skip here has no effect.